readme.md

```diff
@@ -15,37 +15,53 @@ npm install --global boxen-cli
 ```
 $ boxen --help
 
-  Usage
-    $ boxen <string>
-    $ echo <string> | boxen
-
-  Options
-    --border-color      Color of the box border [black|red|green|yellow|blue|magenta|cyan|white|gray]
-    --background-color  Color of the background [black|red|green|yellow|blue|magenta|cyan|white]
-    --border-style      Style of the box border [single|double|round|singleDouble|doubleSingle|classic]
-                        Can also be specified as the characters to use. See below example.
-    --dim-border        Reduce opacity of border
-    --padding           Space between the text and box border
-    --margin            Space around the box
-    --center            Center the box
-    --align             Align the text [left|center|right] (Default: left)
-    --width             Set a fixed width for the box
-
-  Examples
-    $ boxen I ❤ unicorns
-    ┌────────────┐
-    │I ❤ unicorns│
-    └────────────┘
-
-    $ boxen --border-style=doubleSingle …like everyone
-    ╒══════════════╕
-    │…like everyone│
-    ╘══════════════╛
-
-    $ boxen --border-style='1234-~|║' ASCII ftw!
-    1----------2
-    |ASCII ftw!║
-    3~~~~~~~~~~4
+	Usage
```

Comment:
Incorrect indentation on the new content here. It should be exactly like in the help output.